pyart/retrieve/tests/compare_kdp_proc.py

```diff
@@ -0,0 +1,192 @@
+""" Unit tests for pyart.retrieve.kdp_proc module. """
```

Comment:
Changed it to 

""" 
Unit tests for pyart.retrieve.kdp_proc module. 
Can be used to test the Vulpiani and Maesaka estimation methods for the ideal
case of a linear psidp profile with specified slope
"""